bridges/TwitterBridge.php

```diff
@@ -266,8 +266,10 @@ public function collectData(){
 			switch($this->queriedContext) {
 			case 'By keyword or hashtag':
 				returnServerError('No results for this query.');
+				// fall-through
```

Comment:
The comment is required by PSR2. In this case we know that `returnServerError` actually throws an exception.